pages/hardware/devices.md

```diff
@@ -6,126 +6,86 @@ title: Supported Devices
+![Humming Board](/img/hummingBoard.png)
 
-[link](http://boundarydevices.com/product/nitrogen6x-board-imx6-arm-cortex-a9-sbc/)
+### [Nitrogen6x by Boundary Devices](http://boundarydevices.com/product/nitrogen6x-board-imx6-arm-cortex-a9-sbc/)
 
 ![Nitrogen6x](/img/nitrogen6x.jpg)
 
-### NXP/Freescale i.MX6 Sabre Lite
-
-[link](http://boundarydevices.com/product/sabre-lite-imx6-sbc/)
+### [ODROID-C1+](http://www.hardkernel.com/main/products/prdt_info.php?g_code=G143703355573)
 
-![i.MX6 Sabre](/img/iMX6-sabre-lite.jpg)
+![ODROID-C1+](/img/odroidC1Board.jpg)
 
-### Odroid C1+
-
-[link](http://www.hardkernel.com/main/products/prdt_info.php?g_code=G143703355573)
-
-![Odroid C1+](/img/odroidC1Board.jpg)
-
-### ODROID XU4
-
-[link](http://www.hardkernel.com/main/products/prdt_info.php?g_code=G143452239825&tab_idx=1)
+### [ODROID-XU4](http://www.hardkernel.com/main/products/prdt_info.php?g_code=G143452239825&tab_idx=1)
 
 ![ODROID XU4](/img/odroidXUBoard.jpg)
 
-### Zynq ZC702 from Xilinx
-
-[link](http://www.xilinx.com/products/boards-and-kits/ek-z7-zc702-g.html)
-
-![Zynq board](/img/zynq.jpg)
-
-### VIA VAB-820
+### [Intel NUC](http://www.intel.com/content/www/us/en/nuc/overview.html)
 
-[link](http://www.viatech.com/en/boards/pico-itx/vab-820/)
-
-![VIA VAB-820](/img/viaVab820.jpg)
-
-### Intel NUC
+![NUC](/img/nuc.jpg)
 
-[link](http://www.intel.com/content/www/us/en/nuc/overview.html)
+### [Intel Edison](http://www.intel.com/content/www/us/en/do-it-yourself/edison.html)
 
-![NUC](/img/nuc.jpg)
+![Intel Edison](/img/edison.jpg)
 
 ## Beta Support
 
-### BeagleBone Green Wireless
-
-[link](http://www.seeedstudio.com/wiki/SeeedStudio_BeagleBone_Green_Wireless)
+### [Variscite DART-6UL](http://www.variscite.com/products/system-on-module-som/cortex-a7/dart-6ul-freescale-imx-6ul)
 
-![BeagleBone Gren Wireless](/img/beaglebone-green-wireless.jpg)
+![DART-6UL](/img/dart6ul.jpg)
 
-### Samsung Artik 5
+### [NVIDIA Jetson TX2](https://developer.nvidia.com/embedded/buy/jetson-tx2)
 
-[link](https://www.artik.io/modules/overview/artik-5/)
+![Jetson TX2](/img/jetson-tx2.png)
 
-![Samsung Artik 5](/img/artik5/samsung-artik-5.jpg)
+### [UP board](http://www.up-board.org/up/)
 
-### Samsung Artik 10
+![UP board](/img/up-board/up-board.png)
 
-[link](https://www.artik.io/modules/overview/artik-10/)
+### [Rushup Kitra 520](https://www.rushup.tech/kitra520/)
 
-![Samsung Artik 10](/img/artik10/samsung-artik-10.jpg)
+![Kitra 520](/img/kitra/520.png)
 
-### Toradex Colibri iMX6DL
+### [Rushup Kitra 710](https://www.rushup.tech/kitra710/)
 
-[link](http://developer.toradex.com/products/colibri-imx6)
+![Kitra 710](/img/kitra/710.png)
 
-![Toradex Colibry iMX6DL](/img/toradex-colibri-imx6dl.jpg)
+### [Samsung Artik 710](https://www.artik.io/modules/artik-710/)
 
-### Technologic TS-4900
+![Samsung Artik 5](/img/artik710/A710.jpg)
 
-[link](https://www.embeddedarm.com/products/TS-4900)
+### [Technologic TS-4900](https://www.embeddedarm.com/products/TS-4900)
```

Comment:
I think we should move the TS-4900 out of "beta", it should be considered fully supported.